packages/sources/universal/src/endpoint/sandbox.ts

```diff
@@ -4,23 +4,31 @@ import {
   Method,
   Requester,
   Validator,
+  Logger,
+  Config,
+  ExecuteWithConfig,
+  InputParameters,
+  AxiosResponse,
 } from '@chainlink/ea-bootstrap'
-import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/ea-bootstrap'
 import { TimestampedRequestSigner } from '../utils/timestampedRequestSigner'
 
 export const supportedEndpoints = ['sandbox']
 
 export interface ResponseSchema {
-  success?: hexstring
-  error?: string
+  success: hexstring
+  error: {
+    name: string
+    message: string
+    details: string
+  }
 }
 
 export const description =
   'This adapter endpoint sends a custom request to a FaaS sandbox to be executed.'
 
 export type TInputParameters = {
-  code: string
-  httpQueries?: HttpQuery[]
+  source: string
+  queries?: HttpQuery[]
   args?: string[]
   secrets?: string
   codeLocation?: 'onchain'
```

Comment:
This field as well as secretsLocation and language are encoded as integers (= 0). You might want to create enums equivalent to the ones in the Contract: https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/dev/ocr2dr/OCR2DR.sol#L15-L28

Same for "HttpVerb" below.